packages/smooth_app/test/plural_translation_test.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('Plural test', () {
+    const List<Locale> locales = AppLocalizations.supportedLocales;
+
+    for (final Locale locale in locales) {
+      late AppLocalizations? appLocalizations;
+
+      testWidgets('$locale plural test ', (WidgetTester tester) async {
+        await tester.pumpWidget(
+          MaterialApp(
+            localizationsDelegates: AppLocalizations.localizationsDelegates,
+            supportedLocales: AppLocalizations.supportedLocales,
+            locale: locale,
+            builder: (BuildContext context, Widget? child) {
+              appLocalizations = AppLocalizations.of(context);
+
+              return const Placeholder();
+            },
+          ),
+        );
+
+        expect(appLocalizations, isNotNull);
+
+        final List<String> minutes = <String>[];
+        final List<String> hours = <String>[];
+        final List<String> days = <String>[];
+        final List<String> weeks = <String>[];
+        final List<String> months = <String>[];
+        final List<String> compare = <String>[];
+
+        for (int i = -1; i < 1001; i++) {
```

Comment:
I don't understand the tests, and I don't even understand why they don't fail.
What I would suggest:
* test that with parameter `1`, we cannot find `1` in the result (as it's supposed to be string-ified into "one" or whatever, depending on the language)
* test that with parameters bigger than 1, we find exactly that number in the translated string

Reply:
These tests would be to restricting as one wrote in in the Crowdin comments:

> [...] in Arabic the sequence is not like English. there should be an allowance to add between (other) and (count). [...]

So we can't force exact translations. This test was only added because someone accidentely also translated the `count`  variable in the plural strings which would of course stop the plural from working, so this just checks if there is a number at any value between -1 and 1000. This isn't a quality check but to see if the variable still works.

Reply:
Oh you mean we cannot force `count` to be a number in the `.arb` files?
Btw what's the point of `-1`?
Please add your previous comments to the code, because the purpose is not obvious at all.

Reply:
Adding comments...
-1 actually has no specific use I just wanted to have a negative number inside in case someone makes a special version of it and it doesn't work (Maybe throws a error). The documentation on these parts of the translation is not very good, so I am not sure what is even possible.